streamfx-fix-library-name.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index acaa87c..a0adab6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1079,7 +1079,7 @@ if(D_PLATFORM_WINDOWS) # Windows Support
+ endif()
+ 
+ # Minimum Dependencies
+-list(APPEND PROJECT_LIBRARIES libobs)
++list(APPEND PROJECT_LIBRARIES obs)
```

Comment:
I'm still fairly sure this link target is incorrect and targets the UI instead of the `libobs` library. StreamFX's build system requires the presence of `LibObsConfig.cmake`, which defines `libobs` not `obs`. The future 27.2.4+ CMakeLists build script also entirely revolves around the existence of libobs, as well as the necessary cmake files.

Reply:
I added this patch before setting the `libobs_SOURCE_DIR` option, and didn't try to build without it afterwards. Indeed, it is not necessary, I removed it.